README.md

```diff
@@ -0,0 +1,118 @@
+# pds.api-client
+Federated PDS API which provides actionable end points standardized
+between the different nodes.
+
+
+This Python package is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
+
+- API version: 0.3.dev
+- Package version: 0.6.1
+- Build package: org.openapitools.codegen.languages.PythonLegacyClientCodegen
+
+## Requirements.
+
+Python 2.7 and 3.4+
```

Comment:
We probably don't want to advertise Python 2.7 support, even if this was just automatically generated